examples/blog-post-metrics/src/components/Dialog.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Paragraph } from '@contentful/forma-36-react-components';
+import { DialogExtensionSDK } from '@contentful/app-sdk';
+
+interface DialogProps {
+  sdk: DialogExtensionSDK;
+}
+
+const Dialog = (props: DialogProps) => {
+  return <Paragraph>Hello Dialog Component</Paragraph>;
+};
+
+export default Dialog;
```

Comment:
Discussion: The tutorial only talks about `sidebar` and `config` locations. Should we delete unrelated components, or keep them in as a better starting point?

Reply:
I had the same conversation with Andi :D I would prefer to leave it here. Its weird to delete files when you just created them and I think they will serve as a good starting point.